svf-llvm/tools/LLVM2SVF/llvm2svf.cpp

```diff
@@ -61,12 +61,12 @@ int main(int argc, char** argv)
         LLVMModuleSet::preProcessBCs(moduleNameVec);
     }
 
-    SVFModule* svfModule = LLVMModuleSet::buildSVFModule(moduleNameVec);
+    //SVFModule* svfModule = LLVMModuleSet::buildSVFModule(moduleNameVec);
```

Comment:
Why comment this?

Reply:
because unused var would cause compile error.

Reply:
Why unused? Which variable?